frictionless/schema.py

```diff
@@ -302,6 +302,13 @@ def metadata_validate(self):
             if field.builtin:
                 yield from field.metadata_errors
 
+        # Examples
+        for field in [f for f in self.fields if field.example]:
```

Comment:
If you pass a clear `None` in the `example` and you expect a string/integer this will not be checked. Not sure how to approach this, let me know or feel free to edit.